docs/HOWTO.md

```diff
@@ -244,10 +244,10 @@ Then the datasets can be declared in the benchmark definition file as follow:
 - the `folds` attribute is also optional but recommended for those datasets as the default value is `folds=10` (default amount of folds in openml datasets), so if you don't have that many folds for your custom datasets, it is better to declare it explicitly here.
 - Remote files are downloaded to the `input_dir` folder and archives are decompressed there as well, so you may want to change the value of this folder in your [custom config.yaml file](#custom-configuration) or specify it at the command line with the `-i` or `--indir` argument (by default, it points to the `~/.openml/cache` folder).
 
-#### OpenML suites
+#### OpenML studies
```

Comment:
Actually OpenML refers to collections of tasks as a _suite_ since a year or two, see [docs](https://docs.openml.org/benchmark/). A study is a collection of runs (i.e. the results of running algorithms (flows) on a suite (tasks)). It's definitely possible I have used the incorrect naming before.
```suggestion
#### OpenML suites
```

```suggestion
#### OpenML suites
```

Reply:
Ok sorry, I got confused because I had a chat with @ledell recently, and she was referring to studies when I was referring to suites.
Now OpenML UI is not clear as it still refers to `studies` in the menu: https://www.openml.org/search?type=study
so I thought that `study` was the right word...

@ledell FYI

Reply:
Looks like it's not only me that is confused, taken from the docs link you provided:
```
# create the benchmark suite
study = openml.study.create_benchmark_suite( 
    name="MidSize Suite", 
    alias=None,
    description="illustrating how to create a benchmark suite", 
    task_ids=task_ids,
)
```
this example names `study` the result of `create_benchmark_suite` :)

Reply:
another confusing usage:
```python
                    help="The benchmark type to run as defined by default in resources/benchmarks/{benchmark}.yaml, "
                         "a path to a benchmark description file, or an openml suite or task. OpenML references should "
                         "be formatted as 'openml/s/X' and 'openml/t/Y', for studies and tasks respectively. Defaults to `%(default)s`.")
```